libraries/nrpe_installation_package.rb

```diff
@@ -126,7 +126,7 @@ def nagios_plugins
       # @return [String]
       # @api private
       def nrpe_program
-        options.fetch(:program, '/usr/sbin/nrpe')
+        options.fetch(:program, @program)
```

Comment:
Where are we storing that instance variable? The second parameter is the default value.